SonarQube.MSBuild.Tasks/Targets/SonarQube.Integration.targets

```diff
@@ -205,6 +205,19 @@
     </PropertyGroup>
   </Target>
 
+  <Target Name="CleanProjectFolders" AfterTargets="Clean" Condition=" $(SonarQubeTempPath) != '' ">
+    <ItemGroup>
+      <ProjectSpecificOutFiles Include="$(SonarQubeOutputPath)\$(MSBuildProjectName)_*\*.*"/>
```

Comment:
@duncanpMS The idea is to empty out the directories created for each project ($(MSBuildProjectName)_*) before removing them. We want to keep other files in the conf folder, for example, as they are generated in the begin step.